controllers/managedocs_controller.go

```diff
@@ -977,11 +1049,33 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			Key: alertRelabelConfigSecretKey,
 		}
 
-		// prometheus tls -- will be uncommented
-		// tlsConfig := r.prometheus.Spec.Web.TLSConfig
-		// tlsConfig.KeySecret.LocalObjectReference.Name = prometheusServiceSecretName
-		// tlsConfig.Cert.Secret.LocalObjectReference.Name = prometheusServiceSecretName
-		// tlsConfig.ClientCA.ConfigMap.LocalObjectReference.Name = servingCertCABundleConfigMapName
+		volumes := r.prometheus.Spec.Volumes
+		for key := range volumes {
+			volume := &volumes[key]
```

Comment:
- why not just range on volumes and switch on volume.Name rather than multiple conditionals?

Reply:
+1 to replace the `if else` construct with a switch statement